### PR TITLE
chore: New Crowdin updates

### PR DESCRIPTION
**Pull request by lipis**
Each language must be at least **85%** translated in order to appear on Excalidraw. Join us on [Crowdin](https://crowdin.com/project/excalidraw) and help us translate your own language. **Can't find yours yet?** Open an [issue](https://github.com/excalidraw/excalidraw/issues/new) and we'll add it to the list.

| | Flag | Locale | % |
| :--: | :--: | -- | :--: |
| 1 | 🇸🇦 | [**العربية**](https://crowdin.com/translate/excalidraw/10/en-ar) | 💯 |
| 2 | 🇧🇬 | [**Български**](https://crowdin.com/translate/excalidraw/10/en-bg) | 💯 |
| 3 | 🇪🇸 | [**Català**](https://crowdin.com/translate/excalidraw/10/en-ca) | 💯 |
| 4 | 🇩🇪 | [**Deutsch**](https://crowdin.com/translate/excalidraw/10/en-de) | 💯 |
| 5 | 🇬🇷 | [**Ελληνικά**](https://crowdin.com/translate/excalidraw/10/en-el) | 💯 |
| 6 | 🇪🇸 | [**Español**](https://crowdin.com/translate/excalidraw/10/en-es) | 💯 |
| 7 | 🇮🇷 | [**فارسی**](https://crowdin.com/translate/excalidraw/10/en-fa) | 💯 |
| 8 | 🇫🇮 | [**Suomi**](https://crowdin.com/translate/excalidraw/10/en-fi) | 💯 |
| 9 | 🇫🇷 | [**Français**](https://crowdin.com/translate/excalidraw/10/en-fr) | 💯 |
| 10 | 🇮🇱 | [**עברית**](https://crowdin.com/translate/excalidraw/10/en-he) | 💯 |
| 11 | 🇮🇳 | [**हिन्दी**](https://crowdin.com/translate/excalidraw/10/en-hi) | 💯 |
| 12 | 🇭🇺 | [**Magyar**](https://crowdin.com/translate/excalidraw/10/en-hu) | 💯 |
| 13 | 🇮🇩 | [**Bahasa Indonesia**](https://crowdin.com/translate/excalidraw/10/en-id) | 💯 |
| 14 | 🇮🇹 | [**Italiano**](https://crowdin.com/translate/excalidraw/10/en-it) | 💯 |
| 15 | 🇰🇷 | [**한국어**](https://crowdin.com/translate/excalidraw/10/en-ko) | 💯 |
| 16 | 🇳🇴 | [**Norsk bokmål**](https://crowdin.com/translate/excalidraw/10/en-nb) | 💯 |
| 17 | 🇳🇱 | [**Nederlands**](https://crowdin.com/translate/excalidraw/10/en-nl) | 💯 |
| 18 | 🇳🇴 | [**Norsk nynorsk**](https://crowdin.com/translate/excalidraw/10/en-nnno) | 💯 |
| 19 | 🇮🇳 | [**ਪੰਜਾਬੀ**](https://crowdin.com/translate/excalidraw/10/en-pain) | 💯 |
| 20 | 🇵🇱 | [**Polski**](https://crowdin.com/translate/excalidraw/10/en-pl) | 💯 |
| 21 | 🇧🇷 | [**Português Brasileiro**](https://crowdin.com/translate/excalidraw/10/en-ptbr) | 💯 |
| 22 | 🇵🇹 | [**Português**](https://crowdin.com/translate/excalidraw/10/en-pt) | 💯 |
| 23 | 🇷🇴 | [**Română**](https://crowdin.com/translate/excalidraw/10/en-ro) | 💯 |
| 24 | 🇷🇺 | [**Русский**](https://crowdin.com/translate/excalidraw/10/en-ru) | 💯 |
| 25 | 🇸🇰 | [**Slovenčina**](https://crowdin.com/translate/excalidraw/10/en-sk) | 💯 |
| 26 | 🇸🇪 | [**Svenska**](https://crowdin.com/translate/excalidraw/10/en-sv) | 💯 |
| 27 | 🇹🇷 | [**Türkçe**](https://crowdin.com/translate/excalidraw/10/en-tr) | 💯 |
| 28 | 🇺🇦 | [**Українська**](https://crowdin.com/translate/excalidraw/10/en-uk) | 💯 |
| 29 | 🇨🇳 | [**简体中文**](https://crowdin.com/translate/excalidraw/10/en-zhcn) | 💯 |
| 30 | 🇹🇼 | [**繁體中文**](https://crowdin.com/translate/excalidraw/10/en-zhtw) | 💯 |
| 31 | 🇲🇲 | [**Burmese**](https://crowdin.com/translate/excalidraw/10/en-my) | **93** |
| 32 | 🇯🇵 | [**日本語**](https://crowdin.com/translate/excalidraw/10/en-ja) | **85** |

\* Languages in **bold** are going to appear on production.

### CODE REVIEW

File: src/locales/ar-SA.json [hunk 1]
```diff
@@ -69,18 +70,18 @@
...
     "group": "تحديد مجموعة",
     "ungroup": "إلغاء تحديد مجموعة",
     "collaborators": "المتعاونون",
-    "gridMode": "",
+    "gridMode": "وضع الشبكة",
     "addToLibrary": "أضف إلى المكتبة",
     "removeFromLibrary": "حذف من المكتبة",
     "libraryLoadingMessage": "جارٍ تحميل المكتبة...",
-    "libraries": "",
+    "libraries": "تصفح المكتبات",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:84]: المكتبات

File: src/locales/ar-SA.json [hunk 2]
```diff
@@ -117,10 +118,10 @@
     "redo": "إعادة تنفيذ",
     "roomDialog": "بدء المشاركة الحية",
     "createNewRoom": "إنشاء غرفة جديدة",
-    "fullScreen": "",
-    "darkMode": "",
-    "lightMode": "",
-    "zenMode": "",
+    "fullScreen": "شاشة كاملة",
+    "darkMode": "الوضع المظلم",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:122]: الوضع الداكن

File: src/locales/ar-SA.json (hunk 1 above)
Comment (lipis) [reply src/locales/ar-SA.json:84]: Could you make the updates on https://crowdin.com/project/excalidraw?

File: src/locales/ar-SA.json [hunk 3]
```diff
@@ -221,15 +222,15 @@
...
-    "total": "",
-    "width": ""
+    "angle": "الزاوية",
+    "element": "عنصر",
+    "elements": "العناصر",
+    "height": "الارتفاع",
+    "scene": "المشهد",
+    "selected": "المحدد",
+    "storage": "التخزين",
+    "title": "إحصائيات للمهووسين",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:232]: العنوان

Comment (lipis) [reply src/locales/ar-SA.json:232]: All the updates should be done in Crowdin: https://crowdin.com/project/excalidraw

File: src/locales/ar-SA.json [hunk 4]
```diff
@@ -221,15 +222,15 @@
...
-    "element": "",
-    "elements": "",
-    "height": "",
-    "scene": "",
-    "selected": "",
-    "storage": "",
-    "title": "",
-    "total": "",
-    "width": ""
+    "angle": "الزاوية",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:225]: زاوية

File: src/locales/ar-SA.json [hunk 5]
```diff
@@ -221,15 +222,15 @@
...
+    "element": "عنصر",
+    "elements": "العناصر",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:227]: عناصر

File: src/locales/ar-SA.json [hunk 6]
```diff
@@ -221,15 +222,15 @@
...
+    "height": "الارتفاع",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:228]: ارتفاع

File: src/locales/ar-SA.json [hunk 7]
```diff
@@ -221,15 +222,15 @@
...
+    "scene": "المشهد",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:229]: مشهد

File: src/locales/ar-SA.json [hunk 8]
```diff
@@ -221,15 +222,15 @@
...
+    "selected": "المحدد",
+    "storage": "التخزين",
```
Comment (cozmoc) [inline src/locales/ar-SA.json:231]: مخزن

**Pull request merged**